src/languages/javascript.js

```diff
@@ -6,6 +6,8 @@ Website: https://developer.mozilla.org/en-US/docs/Web/JavaScript
 */
 
 function(hljs) {
+  var TAG_START = /<[A-Za-z0-9\\._:-]+/;
+  var TAG_FINISH = /\/[A-Za-z0-9\\._:-]+>|\/>/;
```

Comment:
Hey guys, React now support fragments which are usually written as `<>...</>`.

So maybe this could be
```js
var TAG_START = /<[A-Za-z0-9\\._:-]*/;
var TAG_FINISH = /\/[A-Za-z0-9\\._:-]*>|\/>/;
```

What do you think? Do you see any problem?

Reply:
I think I need an actual real example of what you're talking about.

Reply:
Well now if that ain't just ridiculous. :-)

https://reactjs.org/docs/fragments.html#short-syntax

Could you open an actual issue for this, please?

Reply:
Sure: https://github.com/highlightjs/highlight.js/issues/2327